webpack.config.js

```diff
@@ -21,7 +49,14 @@ module.exports = {
       // both options are optional
       filename: "[name].css",
       chunkFilename: "[id].css"
+    }),
+    new HtmlWebpackPlugin({
+      // generate html file without manually inject .js bundle
+      template: "index.html"
     })
   ],
-  watch: process.env.NODE_ENV !== "production" ? true : false
+  devServer: {
+    contentBase: path.join(__dirname, "dist"),
+    compress: true
```

Comment:
Why do you enable gzip compression for local development?

Reply:
Don't have to...